src/libs/Navigation/AppNavigator/BaseDrawerNavigator.js

```diff
@@ -49,6 +49,18 @@ class BaseDrawerNavigator extends Component {
         };
     }
 
+    componentDidUpdate(prevProps) {
+        if (prevProps.isSmallScreenWidth === this.props.isSmallScreenWidth) {
+            return;
+        }
+        // eslint-disable-next-line react/no-did-update-set-state
+        this.setState({
+            defaultStatus: Navigation.getDefaultDrawerState(
```

Comment:
another minor formatting change

```suggestion
            defaultStatus: Navigation.getDefaultDrawerState(this.props.isSmallScreenWidth)
```